css/style.css

```diff
@@ -0,0 +1,2912 @@
+    padding-right: 25px;
+    margin-right: 0;
+  }
+
+  .middle {
+    max-width: 880px;
+    margin-top: 30px;
+  }
+
+  .middle__inner {
+    align-content: space-between;
+    background: url(../img/tesla-fon-ipad-2.png) no-repeat;
+    background-size: 880px auto;
+  }
+
+  .middle__fon {
+    display: flex;
+    display: inline-block;
+  }
+
+  .middle__title {
+    margin-top: 11px;
+    margin-left: 434px;
+  }
+
+  .middle__info {
+    margin-top: 400px;
+    max-width: 880px;
+    height: 490px;
+    margin-top: 405px;
+  }
+
+  .middle__info-row {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .middle__content {
+    margin-left: 0;
+  }
+
+  .middle__row {
+    margin-right: 0;
+    padding-right: 30px;
+  }
+
+  .middle__info-title {
+    margin-top: 57px;
+    margin-left: 32px;
+  }
+
+  .middle__info-text {
+    width: 320px;
+    margin-right: 0;
+    margin-top: 22px;
+    margin-left: 32px;
+  }
+
+  .middle__row {
+    margin-right: 70px;
+    margin-top: 33px;
+    margin-left: 50px;
+  }
+
+  .middle__screen {
+    padding-bottom: 0;
+    padding-top: 8px;
+  }
+
+  .middle__text {
+    padding-bottom: 25px;
+  }
+
+  .middle__btn {
+    margin-left: 152px;
+    margin-top: 17px;
+  }
+
+  .button__btn {
+    display: flex;
+    align-items: center;
+    margin-left: 150px;
+    margin-right: 0;
+    padding-left: 0;
+    margin-top: 15px;
+  }
+
+  .button__btn-img {
+    width: 32px;
+    height: 32px;
+  }
+
+  .button__btn-word {
+    width: 150px;
+    margin-left: 15px;
+    margin-right: 23px;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .button__btn-text {
+    font-size: 16px;
+    line-height: 20px;
+    margin-left: 20px;
+    padding-left: 63px;
+    padding-right: 63px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-position: 95px 68px;
+  }
+    
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-position: 0 75px;
+  }
+
+  .bottom__inner-img {
+    padding-top: 75px;
+  }
+
+  .bottom__info {
+    height: 490px;
+    width: 880px;
+    margin-top: 538px;
+  }
+
+  .bottom__content-stars {
+    display: flex;
+    height: 365px;
+  }
+
+  .bottom__description {
+    margin-right: 0;
+    padding-left: 35px;
+    padding-top: 62px;
+    max-width: 362px;
+    height: 355px;
+  }
+    
+  .bottom__stars {
+    width: 350px;
+    height: 400px;
+    margin-left: 0;
+    margin-top: 5px;
+    padding-left: 50px;
+  }
+
+  .bottom__appraisal-title {
+    padding-left: 0;
+  }
+
+  .bottom__content {
+    width: 292px;
+    padding-left: 0;
+    margin-left: 0;
+  }
+    
+  .stars {
+    padding-right: 0;
+  }
+    
+  .bottom__shield {
+    display: none;
+  }
+
+  .bottom__btn {
+    margin-left: 152px;
+  }
+
+  .button__btn {
+    margin-top: 0;
+    margin-left: 0;
+  }
+
+  .bottom__info-img {
+   display: none;
+   }  
+
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)